plugins/module_utils/podman/podman_container_lib.py

```diff
@@ -369,7 +369,9 @@ def addparam_env(self, c):
         return c
 
     def addparam_env_file(self, c):
-        return c + ['--env-file', self.params['env_file']]
+        for env_file in self.params['env_files']:
```

Comment:
```suggestion
        for env_file in self.params['env_file']:
```
It should be the real arg, not an alias